indexer/services/comlink/__tests__/controllers/api/v4/affiliates-controller.test.ts

```diff
@@ -0,0 +1,23 @@
+import { RequestMethod } from '../../../../src/types';
+import request from 'supertest';
+import { sendRequest } from '../../../helpers/helpers';
+
+describe('test-controller#V4', () => {
+
+  describe('GET /referral_code', () => {
+
```

Comment:
```suggestion
describe('test-controller#V4', () => {
  describe('GET /referral_code', () => {
```
nit: no need for this much white space